experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1174,6 +1174,19 @@ def validate_value(self, value):
                 json.loads(value)
             except Exception as e:
                 raise serializers.ValidationError(f"Invalid JSON: {e.msg}") from e
+
+        def throw_on_float(item):
+            if isinstance(item, (list, tuple)):
+                [throw_on_float(i) for i in item]
+            elif isinstance(item, dict):
+                [throw_on_float(i) for i in item.values()]
```

Comment:
You're still making lists (my original comment was wrong, I updated it to use loops)

Reply:
Technically yes but it's so minor 🤷‍♀️